  <launch>
    <!-- Turtlesim Node-->
    <node pkg="turtlesim" type="turtlesim_node" name="sim"/>
    <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>
    
    
    <!--<node name="tf_turtle_delay" pkg="tf_nodes" type="turtle_tf_delay_listener.py" >
      <param name="child" type="string" value="turtle1_child" />
    </node>-->

    <!--<node name="tf_turtle" pkg="tf_nodes" type="turtle_tf_listener.py" >
      <param name="child" type="string" value="turtle1_child" />
    </node>-->

    <node name="turtle1_child_frame" pkg="tf_nodes" type="fixed_tf_broadcaster.py" />
    
    <node name="turtle1_tf_broadcaster" pkg="tf_nodes" type="turtle_tf_broadcaster.py" respawn="false" output="screen" >
      <param name="turtle" type="string" value="turtle1" />
    </node>
    
    <node name="turtle2_tf_broadcaster" pkg="tf_nodes" type="turtle_tf_broadcaster.py" respawn="false" output="screen" >
      <param name="turtle" type="string" value="turtle2" /> 
    </node>
    

  </launch>
